setup.py

```diff
@@ -1,5 +1,5 @@
 import os
-from setuptools import setup
+from setuptools import setup, find_packages
```

Comment:
yes, it got dropped - my bad. please do and rebase it on the develop branch. also, please put this on a separate line, one module import per line. thank you

Reply:
Fixed!